### ISSUE TITLE
l2 normalization bug in NetFV

### ISSUE THREAD

**Issue by linrongc** [antoine77340/LOUPE#4]
`the process of intra normalization in NetFV

        fv2 = tf.reshape(fv2,[-1,self.cluster_size*self.feature_size])      
        fv2 = tf.nn.l2_normalize(fv2,1)
        fv2 = tf.reshape(fv2,[-1,self.cluster_size*self.feature_size])
        fv2 = tf.nn.l2_normalize(fv2,1)`

should remove the first reshape?
